core/src/main/resources/hudson/PluginManager/advanced.jelly

```diff
@@ -38,21 +38,6 @@ THE SOFTWARE.
           <l:app-bar title="${%Advanced settings}"/>
 
           <div class="jenkins-form">
-            <section class="jenkins-section jenkins-!-margin-bottom-5">
```

Comment:
do we want to put a notice here for a period of time to say it's moved?

Reply:
👍 from me. I did that when we moved cloud configuration out of the Configure System page and thought that was nice (then we kept it around for too long but that's a separate issue).

Reply:
sure why not notice but what sort of style/color etc.. (fancy icon? :))

Reply:
just the `-info` notification banner I think, (same type as admin monitors have but with just -info)

Reply:
done.